tests/test_app/resources/locales/fr/default.po

```diff
@@ -0,0 +1,5 @@
+msgid "{0} apple"
+msgid_plural "{0} apples"
+msgstr[0] "{0} pomme"
+msgstr[1] "{0} de pommes"
+msgstr[2] "{0} pommes"
```

Comment:
Should we also have a test covering a `msgid_plural` that doesn't have a `2` index as that would reflect an existing translation.

Reply:
Done